src/components/RestaurantCard.tsx

```diff
@@ -0,0 +1,88 @@
+import { Restaurant } from "@/@types";
+import StarIcon from '@mui/icons-material/Star';
+import StarHalfIcon from '@mui/icons-material/StarHalf';
+import StarBorderIcon from '@mui/icons-material/StarBorder';
+
+import '@/styles/restaurant.scss';
+import useIsMobile from "@/hooks/useIsMobile";
+import { Button } from "./Button";
+
+type RestaurantCardProps = {
+  restaurant: Restaurant;
+};
+
+function Price({price}: {price: string}) {
```

Comment:
Couple of components live inside this component as this will not be used anywhere else